.bouncer.yaml

```diff
@@ -3,4 +3,7 @@ permit:
   - MIT.*
   - Apache.*
   - MPL.*
-  - ISC
\ No newline at end of file
+  - ISC
+ignore-packages:
+  # packageurl-go is released under the MIT license located in the root of the repo at /mit.LICENSE
+  - github.com/package-url/packageurl-go
```

Comment:
Nit: I'd recommend enabling the setting (for GoLand) that ensures final newlines